intel_extension_for_transformers/llm/runtime/graph/core/ne_layers.h

```diff
@@ -260,6 +260,7 @@ NE_API struct ne_tensor* ne_norm(struct ne_context* ctx, struct ne_tensor* a);
 
 NE_API struct ne_tensor* ne_rms_norm(struct ne_context* ctx, struct ne_tensor* a);
 
+NE_API struct ne_tensor* ne_rms_norm_inplace(struct ne_context* ctx, struct ne_tensor* a);
```

Comment:
seems ```rmsnorm``` haven't been supported in jblas epilogue? will we support it? @zhewang1-intc

Reply:
we have no plan to support rmsnorm in epilogue now, it's hard to calculate rmsnorm in epilogue in current parallel strategy ( we split the dst matrix on N-dim).